benchmarks/314.hadoop/hadoop.sh

```diff
@@ -7,34 +7,6 @@
 
 #hdfs_tmp=/tmp/hadoop-${USER}
 
-Osname_Ubuntu=`cat /etc/*release | grep -c "Ubuntu"`
-Osname_CentOS=`cat /etc/*release | grep -c "CentOS"`
-
-if [[ "$#" -ne 2  ]] 
-then
-   echo -e "Usage : The number of command line arguments for hadoop.sh  should be 2 ,mount disk and mount point \n"
-   echo -e   " hadoop.sh <disk> <mount_point> \n"
-   echo -e  "Add the arguments in configuration/test_cases_cfg/common/hadoop/hadoop_run.cfg in command section as below \n"
-   echo -e  "command = if [ -f hadoop_tar.gz ]; then tar -xvf hadoop_tar.gz; rm hadoop_tar.gz; fi; pushd hadoop; ./hadoop.sh <disk> <mount_point>; popd \n"
-   echo -e "example: \n"
-   echo -e "command = if [ -f hadoop_tar.gz ]; then tar -xvf hadoop_tar.gz; rm hadoop_tar.gz; fi; pushd hadoop; ./hadoop.sh /dev/sda /mnt/hadoop ; popd \n"
-
-   exit 1
-fi
-
-
-disk=$( echo $1 | cut -d "/" -f 3 )
-
-echo "Disk used for mounting is  $disk"
-
-
-if [ `lsblk | grep -c "$disk"` == 0 ]
-then
-   echo "The disk specified for hadoop testing $1 is not vailable "
-   exit 1
-fi
-
```

Comment:
Prepare for the environment is must, otherwise the case can't run.
And user can change mount origin disk, 
but not need change the mount target directory.